server.py

```diff
@@ -134,7 +137,11 @@ async def get_phone_script():
             logger.info("Regenerating phone script audio file on demand")
             generate_wav_file() 
     
-    return FileResponse(wav_path, media_type="audio/wav", filename='leetcode_latest.wav')
+    month = datetime.datetime.now().month - 1
+    return {
+        "file": FileResponse(wav_path, media_type="audio/wav", filename='leetcode_latest.wav'),
+        "month": month
+    }
```

Comment:
no need to change this, the phone will get the month by calling the leaderboard api